packages/manager/modules/vps/src/import/helper.service.js

```diff
@@ -0,0 +1,20 @@
+export default class VpsHelperService {
+  static extractConfigurationFromPlanCode(planCode) {
+    const [cores, memory, storage] = planCode.match(/\d+/g);
+    return [parseInt(cores, 10), parseInt(memory, 10), parseInt(storage, 10)];
+  }
+
+  parseRangeConfiguration(rangeFullName) {
+    const [
+      cores,
+      memory,
+      storage,
+    ] = this.constructor.extractConfigurationFromPlanCode(rangeFullName);
```

Comment:
```suggestion
    ] = VpsHelperService.extractConfigurationFromPlanCode(rangeFullName);
```

Reply:
Prefer using constructor as i can bypass eslint warning about not using this inside class method.